docs/source/examples/managed-jobs.rst

```diff
@@ -152,6 +152,39 @@ The :code:`MOUNT` mode in :ref:`SkyPilot bucket mounting <sky-storage>` ensures
 Note that the application code should save program checkpoints periodically and reload those states when the job is restarted.
 This is typically achieved by reloading the latest checkpoint at the beginning of your program.
 
+
+.. _intermediate-bucket:
```

Comment:
We should probably move this section to later in the page, maybe right before `Concept: Jobs Controller`